{
  "comments": [
    {
      "key": {
        "uuid": "AAAATn/+XD8\u003d",
        "filename": "update.py",
        "patchSetId": 2
      },
      "lineNbr": 73,
      "author": {
        "id": 4146
      },
      "writtenOn": "2013-11-26T22:57:40Z",
      "side": 1,
      "message": "Did dest_keys become dest_reqs above? I think this name mismatch is why the integration test is failing.",
      "revId": "a3e753aecf54940e6d5b2d50f05bfd43146593f9",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    }
  ]
}